.eslintrc.js

```diff
@@ -1,103 +0,0 @@
-module.exports = {
```

Comment:
Did you delete this file intentionally? I appreciate eslint. Without linting chaos would ensue is the codebase.

Reply:
I have noticed not all repos use the same rules so it would be nice to standardize.